File: static/code/sample/tethered/gdl-hidial-a.py
# In this sample:
# Update Dial value every second

import time
from DUELink.DUELinkController import DUELinkController

availablePort = DUELinkController.GetConnectionPort()
duelink = DUELinkController(availablePort)

# method
def Dial():
    return duelink.Engine.ExecuteCommand("Dial()")

while True:
    print(f"Value: {Dial()}")
    time.sleep(1)  # Sleep for 1 second

File: static/code/sample/tethered/gdl-hidial-a.cs
// In this sample:
// Update Dial value every second

using System;
using GHIElectronics.DUELink;

var availablePort = DUELinkController.GetConnectionPort();
var duelink = new DUELinkController(availablePort);

float Dial() {    
    return duelink.Engine.ExecuteCommand("Dial()");
}

while (true) {    
    Console.WriteLine($"Value: {Dial()}");
    Thread.Sleep(1000);
}

